app/js/environments/environment.ts

```diff
@@ -0,0 +1,16 @@
+// This file can be replaced during build by using the `fileReplacements` array.
+// `ng build --prod` replaces `environment.ts` with `environment.prod.ts`.
+// The list of file replacements can be found in `angular.json`.
+
+export const environment = {
+  production: false
+};
+
+/*
+ * For easier debugging in development mode, you can import the following file
+ * to ignore zone related error stack frames such as `zone.run`, `zoneDelegate.invokeTask`.
+ *
+ * This import should be commented out in production mode because it will have a negative impact
+ * on performance if an error is thrown.
+ */
+// import 'zone.js/dist/zone-error';  // Included with Angular CLI.
```

Comment:
Embark has its own concept of environments which probably doesn't work well with Angular CLI ones. Also all these comments regarding Angular CLI don't apply in this project as we're inside an Embark project. I think we can remove most of them.

If we can't make Embark's environments easily work with this template, I we should probably remove this altogether. 

I know this is unfortunate as this is a crucial feature of Angular CLI, but I also don't think it makes sense to leave this in if we don't actually take advantage of it.

Reply:
**I agree to remove these environment files.** I added them to get closer to angular CLI projects.

I will  remove `environment.ts`/`environment.prod.ts` and `NormalModuleReplacementPlugin`.

**Firstly, can you share a piece of code to show how to distinguish dev/prod env on runtime?** It's needed in [`app/js/main.ts` before](https://github.com/embark-framework/embark-angular-template/pull/3/files#diff-f81ef9eac7d89b29cf09b1a9a89ab8ceR11) call [enableProdMode](https://angular.io/api/core/enableProdMode).